src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -49,14 +50,19 @@ object HttpGoogleServicesDAO {
   /**
    * first step of OAuth dance: redirect the browser to Google's login page
    */
-  def getGoogleRedirectURI(state: String, approvalPrompt: String = "auto"): String = {
-    flow.newAuthorizationUrl()
-        .setRedirectUri(callbackUri.toString)
-        .setState(state)
-        .setAccessType("offline")   // enables refresh token
-        .setApprovalPrompt(approvalPrompt) // "force" to get a new refresh token
-        .build()
-        // TODO: login hint?
+  def getGoogleRedirectURI(state: String, approvalPrompt: String = "auto", overrideScopes: Option[Seq[String]] = None): String = {
+    val urlBuilder = flow.newAuthorizationUrl()
+      .setRedirectUri(callbackUri.toString)
+      .setState(state)
+      .setAccessType("offline")   // enables refresh token
+      .setApprovalPrompt(approvalPrompt) // "force" to get a new refresh token
+
+    overrideScopes match {
```

Comment:
Standard login: this is `None`

Responding to Rawls requesting more scopes: add those here